contracts/multisig/src/state.rs

```diff
@@ -26,6 +26,7 @@ pub const SIGNING_SESSION_COUNTER: Item<Uint64> = Item::new("signing_session_cou
 pub const SIGNING_SESSIONS: Map<u64, SigningSession> = Map::new("signing_sessions");
 // The keys represent the addresses that can start a signing session.
 pub const AUTHORIZED_CALLERS: Map<&Addr, ChainName> = Map::new("authorized_callers");
+pub const CHAIN_CALLER_PAIRS: Map<(ChainName, Addr), ()> = Map::new("callers_for_chain");
```

Comment:
could you check if (&ChainName, &Addr) work as key as well? Then we wouldn't have to do that much cloning